packages/angular_devkit/build_angular/test/browser/aot_spec_large.ts

```diff
@@ -7,11 +7,9 @@
  */
 
 import { Architect } from '@angular-devkit/architect/src/index2';
```

Comment:
Shouldn't this also change to `@angular-devkit/architect`?

Reply:
Yes.  There are a large number of those in the `build-angular` package (60+ files).  I have a separate PR pending that will cleanup the whole package.

Reply:
SGTM